vendor/bat-native-ledger/src/bat_helper.cc

```diff
@@ -2723,4 +2733,37 @@ static bool ignore_ = false;
     writer.EndObject();
   }
 
+  void saveToJson(JsonWriter& writer,
+                  const ledger::PendingContribution& contribution) {
+    writer.StartObject();
+
+    writer.String("publisher_key");
+    writer.String(contribution.publisher_key.c_str());
+
+    writer.String("amount");
+    writer.Double(contribution.amount);
+
+    writer.String("added_date");
+    writer.Uint64(contribution.added_date);
```

Comment:
uint32

Reply:
fixed definition from 32 to 64. We normally use 64 for timestamp